paramiko/client.py

```diff
@@ -335,6 +336,10 @@ def connect(
                 ``password``, ``key_filename`` and ``allow_agent``) and will
                 trigger an exception if given alongside them.
 
+        :param server_hostkey_name:
```

Comment:
Since this is a simple type, it seems to me we should put that type in the `option` here:

```suggestion
        :param str server_hostkey_name:
```